examples/anime-http/server/auth/trendingAnimeServer.ts

```diff
@@ -0,0 +1,26 @@
+// /* eslint-disable no-undef */
+
+// //@ts-ignore
+// import { Message } from "@asyncapi/glee";
+import axios from "axios"
+
+export async function serverAuth({ headers, callback: done }) {
+  await axios.get("https://jsonplaceholder.typicode.com/todos/1")
+
+  console.log("running in auth file in auth folder")
+
+  console.log("headers", headers["authentication"])
+
+  // console.log("network data", res.data)
+
+  // callback(false, 401, "Unauthorized");
+  done(true)
```

Comment:
When auth fails, the server crashes, it should show a nice error message to console and send `401` respond.